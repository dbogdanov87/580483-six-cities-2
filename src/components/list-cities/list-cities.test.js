import React from "react";
import renderer from "react-test-renderer";
import ListCities from "./list-cities.jsx";

it(`CardOffer is rendered correctly`, () => {

  const div = document.createElement(`div`);
  div.id = `map`;
  document.body.appendChild(div);

  const city = {
    id: 6,
    name: `Dusseldorf`,
    location: {
      latitude: 51.2217200,
      longitude: 6.7761600,
      zoom: 12,
    }
  };
  const tree = renderer.create(
      <ListCities
        activeCity={city}
        cities={[]}
        changeCityClickHandler={jest.fn()}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
