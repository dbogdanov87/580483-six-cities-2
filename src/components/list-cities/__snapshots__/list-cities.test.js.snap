// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardOffer is rendered correctly 1`] = `null`;
